examples/basic/object_detection_image.py

```diff
@@ -0,0 +1,30 @@
+import yaml
```

Comment:
I think for all new Python files we need our standard header with brief file description and DeGurim copyright notice

Reply:
@vlad-nn can you point me to a file with out standard header and copyright notice. I can add file description.

Reply:
#
# <filename>.py: short description
#
# Copyright DeGirum Corporation 2023
# All rights reserved
#
# Detailed description
#

Reply:
Thanks Vlad. All python files are now commented according to the above format.